File: Python/2. Condicionales/Ejercicio #1.py
# 1. Hacer un programa que pida 2 numeros y se de cuenta cual de ellos es par, o si ambos lo son

NUM1 = int(input("Valor #1: "))
NUM2 = int(input("Valor #2: "))

if NUM1%2 == 0 and NUM2%2 == 0:
    print("Ambos son pares")
elif NUM1 % 2 == 0 and NUM2 % 2 != 0:
    print(f"Valor #1 es par")
elif NUM1 % 2 != 0 and NUM2 % 2 == 0:
    print(f"Valor #2 es par")
else:
    print("Ninguno es par")

File: 1. C++/3. Estructuras Condicionales/Ejercicio #1.cpp
//1. Escriba un programa que lea 2 numeros y determine cual es mayor.

#include<iostream>

using namespace std;

int main()
{
    float N1, N2;
    cout<<"Digite el valor #1: ";cin>>N1; // cout<<"Digite el valor #1 y #2: ";cin>>N1>>N2;
    cout<<"Digite el valor #2: ";cin>>N2;

    if(N1 == N2) //Primer condicional
    {
        cout<<"Ambos numero son iguales"<<endl;
    }
    else if(N1>N2)  //Esto nos permite colocar un segundo condicional con respecto al anterior.
    {
        cout<<"El valor 1 es mayor que el 2"<<endl;
    }
    else
    {
        cout<<"El valor 2 es mayor que el valor 1"<<endl;
    }
    return 0;
}